src/main/java/com/powsybl/openloadflow/dc/DcLoadFlowParameters.java

```diff
@@ -7,59 +7,71 @@
 package com.powsybl.openloadflow.dc;
 
 import com.powsybl.loadflow.LoadFlowParameters;
-import com.powsybl.math.matrix.MatrixFactory;
-import com.powsybl.openloadflow.lf.AbstractLoadFlowParameters;
 import com.powsybl.openloadflow.dc.equations.DcEquationSystemCreationParameters;
-import com.powsybl.openloadflow.network.LfNetworkParameters;
+import com.powsybl.openloadflow.lf.AbstractLoadFlowParameters;
 
 import java.util.Objects;
 
 /**
  * @author Geoffroy Jamgotchian {@literal <geoffroy.jamgotchian at rte-france.com>}
  */
-public class DcLoadFlowParameters extends AbstractLoadFlowParameters {
+public class DcLoadFlowParameters extends AbstractLoadFlowParameters<DcLoadFlowParameters> {
 
-    private final DcEquationSystemCreationParameters equationSystemCreationParameters;
+    private DcEquationSystemCreationParameters equationSystemCreationParameters = new DcEquationSystemCreationParameters();
 
-    private final boolean distributedSlack;
+    private boolean distributedSlack = true;
```

Comment:
I think that for `distributedSlack` and `balanceType`, you have to use the default values of `LoadFlowParameters`.

Reply:
Done